### ISSUE TITLE
It's possible to shoot with arrows while wielding lajatang

### ISSUE THREAD

**Issue by Sandman25DCSS** [jeremygurr/dcssca#589]
See title.

> You toss away an arrow of dispersal {=f}.
>  The arrow of dispersal {=f} hits the death yak (2).
>  The death yak gores you (7)!

**Comment by yrmvgh** [comment]
You can throw bread normally too, "toss away" is not a real shoot

**Comment by Sandman25DCSS** [comment]
Why is it in the game? I forgot to switch to longbow and then was able to shoot.

**Comment by jeremygurr** [comment]
Yeah I don't like that. I'm not a fan of the bread throwing either. It creates tricky edge cases that add little to the game I think.
